src/pages/register-a-beacon/additional-beacon-use.tsx

```diff
@@ -58,13 +60,28 @@ const AdditionalBeaconUse: FunctionComponent<AdditionalBeaconUseProps> = ({
 
               {uses.length > 0 && (
                 <>
-                  {uses.map((use, index) => (
-                    <BeaconUseSection
-                      index={index}
-                      use={use}
-                      key={`row${index}`}
-                    />
-                  ))}
+                  {uses.map((use, index) => {
+                    // Prompt the user to confirm before deleting their use
+                    const action = "delete your " + prettyUseName(use) + " use";
+                    const consequences =
+                      "You will have the opportunity to review this change at the end.";
+                    const yes =
+                      ActionURLs.deleteCachedUse + "?useIndex=" + index;
+                    const no = PageURLs.additionalUse + "?useIndex=" + index;
+                    const deleteUri =
+                      PageURLs.areYouSure +
+                      buildAreYouSureQuery(action, yes, no, consequences);
```

Comment:
Don't want to hide things away in functions far away from where it's used, but I wonder if that could be good here? Instead of the comment? 🤔 Not 💯 sure though.